### PR TITLE
Server

### PR DESCRIPTION
**Pull request by BukkyOmo**
#### What does this PR do?
This PR:
* Merges my server branch into develop

#### Description of Task to be completed
* To start work on the database, I had to merge my server branch into develop
* I git pulled develop into my server branch
*Then git push my server branch which has now been updated with files from my develop branch
*Merge into develop

#### Any background context you want to provide?
No

#### What are the relevant pivotal tracker stories?(if applicable)
#163038938

### CODE REVIEW

File: app/v1/controllers/question.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+				message: true,
+				data: [{
+					user: user.id,
+					meetup: meetupId,
+					title: request.body.title,
+					content: request.body.content
+				}]
+			});
+		}
+		return response.status(404).json(
```
Comment (robocopkaka) [inline app/v1/controllers/question.js]: Prefer `400` (bad request) or `500` (Internal server error) for POST requests

File: app/v1/middleware/meetupValidation.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	 * @param {object} next
+	 * @returns {object}- status code and error message or next()
+	 * @memberof meetupValidator
+	 */
+	static createMeetupValidator(request, response, next) {
+		request.check('topic', 'Topic is required').notEmpty();
+		request.check('topic', 'Topic must be a string').isString();
+		request.check('location', 'Location is required').notEmpty();
+		request.check('location', 'Location must be a string').isString();
+		request.check('happeningOn', 'Happening Date is required').notEmpty();
```
Comment (robocopkaka) [inline app/v1/middleware/meetupValidation.js]: Think `Date is required` is preferable to `Happening Date is required`

Comment (BukkyOmo) [reply app/v1/middleware/meetupValidation.js]: okay

File: app/v1/controllers/question.js (hunk 1 above)
Comment (BukkyOmo) [reply app/v1/controllers/question.js]: okay, will implement it now

**Pull request merged**